packages/core/src/html/MdAttributeRenderer.ts

```diff
@@ -75,10 +75,23 @@ export class MdAttributeRenderer {
     return hasAttribute;
   }
 
-  processPopoverAttributes(node: MbNode) {
-    if (!this.hasSlotOverridingAttribute(node, 'header')) {
-      this.processAttributeWithoutOverride(node, 'header', true);
+  /**
+   * Checks if there is a pre-existing slot for the attribute.
+   * If there is a slot, it deletes the attribute and logs a warning.
+   * If there is no slot, it processes the markdown attribute.
+   * @param node Element to process
+   * @param attribute Attribute name to process
+   * @param isInline Whether to process the attribute with only inline markdown-it rules
+   * @param slotName Name attribute of the <slot> element to insert, which defaults to the attribute name
+   */
+  processSlotAttribute(node: MbNode, attribute: string, isInline: boolean, slotName = attribute) {
```

Comment:
I'm not entirely happy with the name of the method but it's the best I can come up with. Some other alternatives I've considered are:
* `processSlottableAttribute`
* `processAttributeCanBeOverridden`
* `processOverridableAttribute`
* `processAttributeWithSlot`

Let me know your thoughts thanks.

Reply:
I think processSlotAttribute is ok!